### PR TITLE
Blackduck license scan integration, part 2

### PR DESCRIPTION
**Pull request by FireyFly**
Turns out the Github Actions secrets are only accessible to private repositories with our configuration. So, move over to using CircleCI instead for the Blackduck scans, even if it means maintaining two CI configs (and that the CircleCI one doesn't work for public forks).

No effort has been made to gate the CircleCI config in ways similar to the Github Actions one (just succeed if there's no token defined), reasoning instead that public repos are expected to just remove/not use the CircleCI integration.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+version: 2.1
+orbs:
+  python: circleci/python@2.0.3
+  tmh-blackduck: mobilityhouse/blackduck-runner-orb@1
+
+aliases:
+  - &base_blackduck_main
+    name: "blackduck automatic"
+    context:
+      - tmh
```
Comment (thomas-sieren-tmh) [inline .circleci/config.yml:10]: @FireyFly I think the `tmh` context here is not needed, as the tool does not have any private dependencies

Comment (thomas-sieren-tmh) [inline .circleci/config.yml:10]:
```suggestion
```

Comment (FireyFly) [reply .circleci/config.yml]: You're right, removed that context since it's superfluous.

**Review by thomas-sieren-tmh — APPROVED** [review]
would be interesting to know if other teams also take this route (using CircleCi in addition to github actions, for open source repos).

**Review by thomas-sieren-tmh — APPROVED** [review]

**Pull request merged**